### PR TITLE
Partial fix for REDISIO-186: RedisGraph broken links

### PR DESCRIPTION
**Pull request by nermiller**
This PR fixes RedisGraph broken links.

### CODE REVIEW

File: docs/docs/_index.md [hunk 1]
```diff
@@ -125,18 +125,18 @@ reply = r.graph("social").query("MATCH (r:Rider)-[:rides]->(t:Team {name:'Ducati
...
 
-The full list and links can be found on [the Clients page](/redisgraph/clients).
+The full list and links can be found on the [Clients](/docs/clients) page.
 
 ## Data import
 
 The RedisGraph team maintains the [redisgraph-bulk-loader](https://github.com/redisgraph/redisgraph-bulk-loader) for importing new graphs from CSV files.
 
-The data format used by this tool is described in the [GRAPH.BULK implementation details](/graph/design/bulk_spec).
+The data format used by this tool is described in the [GRAPH.BULK implementation details](docs/design/bulk_spec).
```
Comment (banker) [inline docs/docs/_index.md:134]:
```suggestion
The data format used by this tool is described in the [GRAPH.BULK implementation details](/docs/design/bulk_spec).
```

**Review by banker — CHANGES_REQUESTED** [review]

**Comment by nermiller** [comment]
@itamarhaber please review and merge. Thanks!

File: docs/docs/design/client_spec.md [hunk 2]
```diff
@@ -270,5 +268,5 @@ In the case of an IndexError, it issues a procedure call to fully refresh its la
 
 ## Reference clients
 
-All the logic described in this document has been implemented in most of the clients listed in [Client Libraries](clients/#currently-available-libraries). Among these, `node-redis`, `redis-py` and `jedis` are currently the most sophisticated.
+All the logic described in this document has been implemented in most of the clients listed in [Client Libraries](/docs/clients). Among these, `node-redis`, `redis-py` and `jedis` are currently the most sophisticated.
```
[the remark this answers is not in the document]
Comment (nermiller) [reply docs/docs/design/client_spec.md:271]: let's try it. it may be (../clients). hard to tell without the preview.

**Comment by nermiller** [comment]
Hi, @itamarhaber, one of the tests failed. Let me know if there is anything I need to do. Thanks!

**Review by AviAvni — APPROVED** [review]

**Pull request merged**